### PR TITLE
Fix/missing keys warning

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
No Missing keys warning should be display in the console when customising some of the keys for a supported language. It fixes the following issue #1240 

# Solution
* Only return warning for unsupported language locale tags
* Refactor code
* Fix typos
* Fix minor dependency vulnerabilities

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by ethan605 — APPROVED** [review]
LGTM!

File: dist/demo.LICENSES.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/** @license React v16.13.1
+/** @license React v16.13.0
```
Comment (kopijunkie) [inline dist/demo.LICENSES.txt:1]: the React version in the licence file here & below seems to have reverted to an older version on the PR branch. 🤔

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1279-pr-onfido-sdk-ui-onfido.surge.sh

File: licenses.json [hunk 2]
```diff
@@ -8774,7 +8900,7 @@
     "publisher": "Web SDK Customer Support",
     "email": "web-sdk@onfido.com",
     "url": "https://github.com/onfido",
-    "path": "/home/ethanify/Work/onfido-sdk-ui",
+    "path": "/Users/stefaniacardenas/Desktop/Code/onfido/onfido-sdk-ui",
```
Comment (kopijunkie) [inline licenses.json:8903]: Out of scope for this PR's work, but only just noticed this local dev directory path overwrite happening here. 😬

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1279-pr-onfido-sdk-ui-onfido.surge.sh

**Review by kopijunkie — APPROVED** [review]
Couple of minor, unrelated issues but otherwise LGTM

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1279-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1279-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**